runtime.c

```diff
@@ -209,15 +209,24 @@ print_help()
 {
 // TODO:   "--appimage-list                 List content from embedded filesystem image\n"
     printf("AppImage options:\n\n"
-           "--appimage-extract              Extract content from embedded filesystem image\n"
-           "--appimage-help                 Print this help\n"
-           "--appimage-mount                Mount embedded filesystem image and\n"
-           "                                print mount point and wait for kill with Ctrl-C\n"
-           "--appimage-offset               Print byte offset to start of\n"
-           "                                embedded filesystem image\n"
-           "--appimage-signature            Print digital signature embedded in AppImage\n"
-           "--appimage-updateinfo[rmation]  Print update info embedded in AppImage\n"
-           "--appimage-version              Print version of AppImageKit\n"
+           "  --appimage-extract              Extract content from embedded filesystem image\n"
+           "  --appimage-help                 Print this help\n"
+           "  --appimage-mount                Mount embedded filesystem image and\n"
+           "                                  print mount point and wait for kill with Ctrl-C\n"
+           "  --appimage-offset               Print byte offset to start of\n"
+           "                                  embedded filesystem image\n"
+           "  --appimage-portable-home        Create a portable home folder to use as $HOME\n"
+           "  --appimage-portable-config      Create a portable config folder to use as $XDG_CONFIG_HOME\n"
+           "  --appimage-signature            Print digital signature embedded in AppImage\n"
+           "  --appimage-updateinfo[rmation]  Print update info embedded in AppImage\n"
+           "  --appimage-version              Print version of AppImageKit\n\n"
+           ""
```

Comment:
(Just asking) would it make sense to move one of the `\n` to this empty line? I guess it was added to highlight the empty 
line in the output, but it'd look more consistent if there was a `\n` here, too, instead of having two of them in the previous line.